themes/smi/layouts/index.html

```diff
@@ -1,129 +1,156 @@
-          <a href="https://solo.io" target="_blank" title="Solo.io"><img src="../img/logo-soloio.png" /></a>
-          <a href="https://redhat.com/" target="_blank" title="Red Hat"><img src="../img/logo-redhat.png" /></a>
-          <a href="https://rancher.com/" target="_blank" title="Rancher"><img src="../img/logo-rancher.png" /></a>
-          <a href="https://docker.com/" target="_blank" title="Docker"><img src="../img/logo-docker.png" /></a>
-          <a href="https://layer5.io/" target="_blank" title="Layer5"><img src="../img/logo-layer5.png" /></a>
-          <a href="https://weave.works/" target="_blank" title="Weaveworks"><img src="../img/logo-weaveworks.png" /></a>
-
-          <a href="https://aspenmesh.io/" target="_blank" title="Aspen Mesh"><img src="../img/logo-aspenmesh.png" /></a>
-          <a href="https://pivotal.io/" target="_blank" title="Pivotal"><img src="../img/logo-pivotal.png" /></a>
-          <a href="https://vmware.com/" target="_blank" title="VMware"><img src="../img/logo-vmware.png" /></a>
-          <a href="https://kinvolk.io/" target="_blank" title="Kinvolk"><img src="../img/logo-kinvolk.png" /></a>
-          <!-- <div class="partner hide-for-small-only">&nbsp;</div> -->
-          <a href="https://canonical.com/" target="_blank" title="Canonical"><img src="../img/logo-canonical.png" /></a>
-          <a href="https://kubecost.com/" target="_blank" title="Kubecost"><img src="../img/logo-kubecost.png" /></a>
-          <!-- <div class="partner hide-for-small-only">&nbsp;</div> -->
+
+    <div class="row">
+      <div class="small-12 large-4 columns">&nbsp;</div>
+      <div class="small-12 large-7 columns end">
+        <div class="panel">
+          <h3>Provider Agnostic</h3>
+          <p>The goal of the SMI API is to provide a common, portable set of service mesh APIs which a Kubernetes user can use in a provider agnostic manner. In this way people can define applications that use service mesh technology without tightly binding to any specific implementation.</p>
         </div>
-        <br>
-        <div class="sep"></div>
       </div>
     </div>
 
-    <div class="row row-full" id="communications">
-      <div class="small-12 large-10 large-centered columns">
-        <h3 class="text-center">Communications:</h3>
 
-        <div class="small-12 large-10 large-centered columns">
-          <div class="panel">
-            <!-- <h4>Slack Channel</h4> -->
-            <div class="panel-wrapper small-12 large-8">
-              <p>Sign up for the <a href="http://smi-spec.slack.com/"><strong>SMI Slack</strong></a> for related discussion.</p>
+  </div>
+</div>
+
+<div class="row row-full" id="credits">
+  <h2 class="text-center sep sep-sides">Ecosystem</h2>
+  <div class="small-12 large-10 large-centered columns">
+    <h3 class="text-center sep sep-sides" id="implementations">Implementations</h3>
```

Comment:
@tomkerkhove we'll want to ensure Meshery's inclusion here. Please see https://layer5.io/brand for a logo pack.

Reply:
Added in the next PR push, but noticed that "Download Logo Asset" buttons are broken on https://layer5.io/brand